server/test/flashcard_test.js

```diff
@@ -0,0 +1,325 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import app from "../index.js";
+import Flashcard from "../models/flashcard.js"
+
+//Assertion Style
+chai.should();
+
+chai.use(chaiHttp);
+
+describe('Flashcard API', () => {
+
+    // SET UP
+    describe("SET UP", () => {
+        before("Empty the DB", (done) => {
+            Flashcard.deleteMany({}, (err) => {
```

Comment:
I'm curious of how does this setup works? Seems like you're using your main db and doing `deleteMany()` with no conditions. Seems like this clears all flashcards in the db and creates new flashcards for test purposes? When users start storing their flashcards, will this be an issue?

Reply:
I pretended this would be performed in a test environment, but will do it in a different way.